src/network/networkd-util.c

```diff
@@ -5,6 +5,7 @@
 #include "escape.h"
 #include "networkd-link.h"
 #include "networkd-util.h"
+#include "networkd-network.h"
```

Comment:
Please sort the headers.

Reply:
After rework `config_parse_link_local_address` the header isn't needed anymore.